### ISSUE TITLE
Bir Compiler: Assertion failures with :optima.test

### ISSUE THREAD

**Issue by kpoeck** [clasp-developers/clasp#1075]
`(ql:quickload :optima.test :verbose nil)`
and
`(eos:run! 'optima.test::optima-test)``
->
```lisp
........; caught ERROR:
;   The assertion (AND
;                  (CLEAVIR-SET:EMPTY-SET-P
;                   (CLEAVIR-BIR:PREDECESSORS CLEAVIR-BIR:IBLOCK))
;                  (CLEAVIR-SET:EMPTY-SET-P
;                   (CLEAVIR-BIR:ENTRANCES CLEAVIR-BIR:IBLOCK))) failed
X; 
; compilation unit aborted
;   caught 1 ERROR condition
.......................................................................................................................................; caught ERROR:
;   The assertion (AND
;                  (CLEAVIR-SET:EMPTY-SET-P
;                   (CLEAVIR-BIR:PREDECESSORS CLEAVIR-BIR:IBLOCK))
;                  (CLEAVIR-SET:EMPTY-SET-P
;                   (CLEAVIR-BIR:ENTRANCES CLEAVIR-BIR:IBLOCK))) failed
X; 
; compilation unit aborted
;   caught 1 ERROR condition
..............................; caught ERROR:
;   The assertion (AND
;                  (CLEAVIR-SET:EMPTY-SET-P
;                   (CLEAVIR-BIR:PREDECESSORS CLEAVIR-BIR:IBLOCK))
;                  (CLEAVIR-SET:EMPTY-SET-P
;                   (CLEAVIR-BIR:ENTRANCES CLEAVIR-BIR:IBLOCK))) failed
X; 
; compilation unit aborted
;   caught 1 ERROR condition
...
 Did 179 checks.
    Pass: 176 (98%)
    Skip: 0 ( 0%)
    Fail: 3 ( 1%)

 Failure Details:
 --------------------------------
 MULTIPLE-VALUE-EMATCH []: 
      Unexpected Error: #<EXT:ASSERT-ERROR>
The assertion (AND
               (CLEAVIR-SET:EMPTY-SET-P
                (CLEAVIR-BIR:PREDECESSORS CLEAVIR-BIR:IBLOCK))
               (CLEAVIR-SET:EMPTY-SET-P
                (CLEAVIR-BIR:ENTRANCES CLEAVIR-BIR:IBLOCK))) failed..
 --------------------------------
 --------------------------------
 MATCH []: 
      Unexpected Error: #<EXT:ASSERT-ERROR>
The assertion (AND
               (CLEAVIR-SET:EMPTY-SET-P
                (CLEAVIR-BIR:PREDECESSORS CLEAVIR-BIR:IBLOCK))
               (CLEAVIR-SET:EMPTY-SET-P
                (CLEAVIR-BIR:ENTRANCES CLEAVIR-BIR:IBLOCK))) failed..
 --------------------------------
 --------------------------------
 EMATCH []: 
      Unexpected Error: #<EXT:ASSERT-ERROR>
The assertion (AND
               (CLEAVIR-SET:EMPTY-SET-P
                (CLEAVIR-BIR:PREDECESSORS CLEAVIR-BIR:IBLOCK))
               (CLEAVIR-SET:EMPTY-SET-P
                (CLEAVIR-BIR:ENTRANCES CLEAVIR-BIR:IBLOCK))) failed..
 --------------------------------

NIL

````

**Comment by Bike** [comment]
Should be fixed by s-expressionists/Cleavir@296a817326908df14fe0b1c29c503c4586393b65

**Comment by kpoeck** [comment]
Retested and fixed
